source/extensions/filters/network/redis_proxy/conn_pool_impl.h

```diff
@@ -93,6 +99,7 @@ class InstanceImpl : public Instance {
     std::unordered_map<Upstream::HostConstSharedPtr, ThreadLocalActiveClientPtr> client_map_;
     Envoy::Common::CallbackHandle* host_set_member_update_cb_handle_{};
     std::unordered_map<std::string, Upstream::HostConstSharedPtr> host_address_map_;
+    std::string auth_password_;
```

Comment:
const

Reply:
ok ... but this will require explict const_cast usage in the constructor to set it initially